src/styles/colors.ts

```diff
@@ -3,14 +3,13 @@ import * as g10 from '@carbon/themes/src/g10.js';
 import * as g100 from '@carbon/themes/src/g100.js';
 import { logIssue } from '../helpers';
 
-/** Theme choices available */
 export type ThemeChoices = 'light' | 'dark';
-/** Theme definition */
-export type ThemeDefinition = { [key: string]: string };
+export type ThemeDefinition = Record<string, string>;
+export type ThemeOverride = Partial<ThemeDefinition>;
```

Comment:
Override is, according to docs, partial of theme map... it doesn't a lot like this type-wise, but at least it does communicate clearly the intention.

Sidenote: I am now generating the full type map in my project for my sanity.